elastalert/__init__.py

```diff
@@ -42,7 +44,12 @@ def es_version(self):
         Returns the reported version from the Elasticsearch server.
         """
         if self._es_version is None:
-            self._es_version = self.info()['version']['number']
+            for retry in range(3):
+                try:
+                    self._es_version = self.info()['version']['number']
+                    break
+                except TransportError:
+                    time.sleep(3)
```

Comment:
if it fails 3 times in a row, you'll leave the version as None, is that going to cause issues downstream? should you raise instead when the retries are exhausted?

Reply:
Oops, good call. Third one should just re-raise. It would have caused a "Nonetype has no method split" type error I believe.